tools/cli/commands/watch.js

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Coherent Labs AD. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+/* eslint-disable max-lines-per-function */

```

Comment:
Can be removed.

Reply:
Why?

Reply:
There is no function that exceeds the max-line-per-function limit.